e2e/tests/chat.spec.ts

```
@@ -55,15 +55,43 @@ test.describe("Chat tests", () => {
+    expect(identicalResponses).toBe(true);
+  });
+  
+  test(`Validate UI response matches API response for a given question in chat`, async () => {
+    const api = new ApiCalls();
+    const apiResponse = await api.askQuestion(GRAPHRAG_SDK, Node_Question);
+    const chat = await browser.createNewPage(CodeGraph, urls.baseUrl);
+    await chat.selectGraph(GRAPHRAG_SDK);
+    await delay(3000);
+    await chat.sendMessage(Node_Question);
+    const uiResponse = await chat.getTextInLastChatElement();
+    const number = uiResponse.match(/\d+/g)?.[0]!;
```

Biome (2.4.4)

[error] Forbidden non-null assertion after optional chaining.

(lint/suspicious/noNonNullAssertedOptionalChain)